util/helm/tags.go

```diff
@@ -0,0 +1,42 @@
+package helm
+
+import (
+    "fmt"
+
+    log "github.com/sirupsen/logrus"
+
+    "github.com/Masterminds/semver/v3"
+)
+
+type TagsList struct {
+	Tags []string
+}
+
+func (t TagsList) MaxVersion(constraints *semver.Constraints) (*semver.Version, error) {
```

Comment:
Probably would be good to put some unit tests on this one.

Reply:
note this is very similar to `Entries.MaxVersion` (index.go) - I would have merged them together, but I don't know how to do it properly. probably it makes sense to keep them separated though, so I'll add the unit test